WordPress/Classes/ViewRelated/Reader/Cards/ReaderPostCell.swift

```diff
@@ -360,6 +370,17 @@ private final class ReaderPostCellView: UIView {
         }
     }
 
+    private func configureSeenCheckmarkIfNeeded() {
+        guard !isSeenCheckmarkConfigured else { return }
+        isSeenCheckmarkConfigured = true
```

Comment:
Out of curiosity, why is a special flag used here? If the idea is to prevent repeated changes to `image`, maybe a lazy var is more appropriate?

Reply:
It's a bit cumbersome, but the view is created eagerly during the setup:

```
let headerView = UIStackView(alignment: .center, [buttonAuthor, dot, timeLabel, seenCheckmark])
```

Ideally, I should create it lazily and add it lazily to the stack view, but I felt it was simpler this way.

Reply:
I guess you could make a `lazy var seenCheckmar: UIImage` so that this configuration function can be called repeatedly. Maybe it's personal preference, but I'm not a big fan of boolean flag type of states.